service/service.py

```diff
@@ -89,6 +89,15 @@ def internal_server_error(error):
 def get_suppliers(supplierID):
 	return supplier.find(supplierID)
 
+######################################################################
+# DELETE A SUPPLIER
+######################################################################
+
+@app.route('/suppliers/<string:supplierID>', methods = ['DELETE'])
+def delete_supplier(supplierID):
```

Comment:
Would be better to make the name consistent with others: 'delete_suppliers'

Reply:
> Would be better to make the name consistent with others: 'delete_suppliers'

I'm not sure... Apparently this function call would only delete one supplier since the query input **"supplierID"** is unique.

Could better be **delete_a_supplier**?

Reply:
> > Would be better to make the name consistent with others: 'delete_suppliers'
> 
> I'm not sure... Apparently this function call would only delete one supplier since the query input **"supplierID"** is unique.
> 
> Could better be **delete_a_supplier**?
> 
> And others need to be changed like **update_a_supplier**, **create_a_supplier** correspondingly?

That works for me.

Reply:
Thanks, has been updated